nikola/plugins/command/plugin.py

```diff
@@ -229,104 +217,84 @@ def do_install(self, url, name, show_install_notes=True):
             utils.makedirs(self.output_dir)
             url = data[name]
             LOGGER.info("Downloading '{0}'".format(url))
-            try:
-                zip_data = requests.get(url).content
-            except requests.exceptions.SSLError:
-                LOGGER.warning("SSL error, using http instead of https (press ^C to abort)")
-                time.sleep(1)
-                url = url.replace('https', 'http', 1)
-                zip_data = requests.get(url).content
+            zip_data = requests.get(url).content
```

Comment:
While I'm glad that this dangerous fallback is no longer there, it might be a good idea to mention it in the changelog (probably even as a security issue).